configure.ac

```diff
@@ -1069,9 +1069,16 @@ dnl check for non-standard math functions
 AC_CHECK_FUNCS([exp10])
 
 dnl pthreads
-AS_IF([test "x$enable_hpcgap" = xyes -o "x$with_julia" != xno ],[
+dnl unconditionally check for pthread flags (and use them), even though we
+dnl only use pthreads if HPC-GAP is enabled. The reason is that if a kernel
+dnl extension uses pthreads (e.g. that of the Semigroups package does it),
+dnl then it may be necessary under certain Linux variants that the
+dnl gap executable is linked with flags like `-pthread`. If it is not, then
+dnl using the kernel extension leads to a runtime exception.
+dnl See also <https://github.com/gap-system/gap/issues/5192>
```

Comment:
I changed the phrasing here a bit and fix typos. If any native speakers would like to double check... :-)

Reply:
Sounds good to me!